File: MyLanguageDB/src/main/java/ru/herooo/mylanguagedb/repositories/wordstatus/WordStatuses.java
package ru.herooo.mylanguagedb.repositories.wordstatus;

public enum WordStatuses {

    ACTIVE(1, "active"),
    UNCLAIMED(2, "unclaimed"),
    NEW(3, "new"),
    BLOCKED(5, "blocked")
    ;

    public final long ID;
    public final String CODE;

    WordStatuses(long id, String title) {
        this.ID = id;
        this.CODE = title;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/api/word_statuses/word_statuses.js
class WordStatus {
    ID;
    CODE;

    constructor(id, code) {
        this.ID = id;
        this.CODE = code;
    }
}

export class WordStatuses {
    ACTIVE = new WordStatus(1, "active");
    UNCLAIMED = new WordStatus(2, "unclaimed");
    NEW = new WordStatus(3, "new");
    BLOCKED = new WordStatus(4, "blocked");
}